historical/amiga

```diff
@@ -7,64 +7,76 @@ copy makefiles/scoptions.000 to opcode_table_generator/scoptions
 cd opcode_table_generator
 
 sc define AMIGA=1 define GB=1 main.c
-slink FROM LIB:c.o main.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
-opcode_table_generator
-delete opcode_table_generator main.o
+sc define AMIGA=1 define GB=1 /opcodes_gb.c
+slink FROM LIB:c.o main.o opcodes_gb.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
+opcode_table_generator /opcodes_gb_tables.c
+delete opcode_table_generator main.o opcodes_gb.o
 
 sc define AMIGA=1 define Z80=1 main.c
-slink FROM LIB:c.o main.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
-opcode_table_generator
-delete opcode_table_generator main.o
+sc define AMIGA=1 define Z80=1 /opcodes_z80.c
+slink FROM LIB:c.o main.o opcodes_z80.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
+opcode_table_generator /opcodes_z80_tables.c
+delete opcode_table_generator main.o opcodes_z80.o
 
 sc define AMIGA=1 define MCS6502=1 main.c
-slink FROM LIB:c.o main.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
-opcode_table_generator
-delete opcode_table_generator main.o
+sc define AMIGA=1 define MCS6502=1 /opcodes_6502.c
+slink FROM LIB:c.o main.o opcodes_6502.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
+opcode_table_generator /opcodes_6502_tables.c
+delete opcode_table_generator main.o opcodes_6502.o
 
 sc define AMIGA=1 define WDC65C02=1 main.c
-slink FROM LIB:c.o main.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
-opcode_table_generator
-delete opcode_table_generator main.o
+sc define AMIGA=1 define WDC65c02=1 /opcodes_65c02.c
+slink FROM LIB:c.o main.o opcodes_65c02.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
+opcode_table_generator /opcodes_65c02_tables.c
+delete opcode_table_generator main.o opcodes_65c02.o
 
 sc define AMIGA=1 define MC6800=1 main.c
-slink FROM LIB:c.o main.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
-opcode_table_generator
-delete opcode_table_generator main.o
+sc define AMIGA=1 define MC6800=1 /opcodes_6800.c
+slink FROM LIB:c.o main.o opcodes_6800.o TO opcode_table_generator LIB LIB:scm.lib LIB:sc.lib LIB:amiga.lib
+opcode_table_generator /opcodes_65c02_tables.c
```

Comment:
Shouldn't this line be

opcode_table_generator /opcodes_6800_tables.c

?

Reply:
Yes, it should.